### ISSUE TITLE
HMR doesn't replace tailwind css

### ISSUE THREAD

**Issue by riywo** [crxjs/chrome-extension-tools#288]
### Build tool

Vite

### Where do you see the problem?

- [X] In the browser
- [ ] In the terminal

### Describe the bug

Related to #274 , but I'm not sure it's the same issue. I'll spend more time to reproduce later and update this description. Sorry for a poor report now.

When I update a tailwind class in a React component, RPCE reloads components and its class name is updated. However, the css embedded by `content_scripts` isn't updated until I restart `vite`. Therefore, the new class isn't accessible even I reload the target page by browser because tailwind postcss? only imports the classes actually used.

By checking Chrome devtools, it looks like triggering some update on the embedded css but the content is still old.

When I access the `vite` server directly, though, the css embedded is updated correctly (the new class is appended). So, I guess the signal is sent from `vite` server but RPCE misses it somewhere. 

I can workaround by using `nodemon` to restart the `vite` server whenever files are changed, but it's slower than HMR.

### Reproduction

TODO

### Logs

```shell
Will update once created a repro.
```

### System Info

```shell
Will update once created a repro.
```

### Severity

annoyance

**Comment by liaoliao666** [comment]
I have also this issue, its a pain point to me. hope it gets fixed soon.

**Comment by jacksteamdev** [comment]
This should be fixed in v1.0.6 :rocket: [@crxjs/vite-plugin - npm](https://www.npmjs.com/package/@crxjs/vite-plugin)
